### PR TITLE
Upstream pull time babyyyyy

### PR DESCRIPTION
**Pull request by Merlin1230**
time to pick and choose commits

### CODE REVIEW

File: SQL/beestation_schema.sql [hunk 1]
```diff
@@ -532,6 +532,18 @@ CREATE TABLE IF NOT EXISTS `SS13_stickyban_matched_ip` (
   PRIMARY KEY (`stickyban`,`matched_ip`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
 
+--
```
Comment (nednaZ) [inline SQL/beestation_schema.sql:535]: Is there anything we will need to handle with our own database before taking on their new achievement system?

**Comment by nednaZ** [comment]
Looks good as long as our database can handle the new achievement system immediately

**Comment by Merlin1230** [comment]
Zamu will have to make the new tables, but thats pretty easy

**Comment by nednaZ** [comment]
After going through a second time now...

#5955 (Beecoin greentext) seems to be disagreed with for us.

#5796 I personally disagree with, RCDs are useful and needed. It's only a crew nerf and a CE buff.

#5851 No. No please. It's just going to be endless nya~ spam.

#5574 I want to look at this one more before we commit to it or not. Messes with Teratomas so that may need to be modified on our end.

**Comment by Merlin1230** [comment]
> 5955(Beecoin greentext) seems to be disagreed with for us.

I agree with it, but I can't remember what ook's opinion was on the matter. I'll leave the final decision to him

> 5796(RCDs)I personally disagree with, RCDs are useful and needed. It's only a crew nerf and a CE buff.

actually, I agree with this PR, it does trivialize building, and I'm pretty sure you can still make them in the engi fab once the tech is research. TBH, I didn't even know engis GOT RCDs. Besides, you can still build everything without an RCD.

> 5851(Tilde fixes) No. No please. It's just going to be endless nya~ spam.

As a felinid player I can tell you that there is nothing wrong with this. Besides, is our community GOING to spam nya~, and is there a problem with them spamming it?

>5574(Viro updates) Generally good, but:
*Self-respiration no longer causes farting.
*Teratomas cannot spawn from a brain-dead person.
*Wizarditus no longer gives real clothing.

honestly those changes seem fine to me

**Comment by lmenvs** [comment]
> 5955(Beecoin greentext) seems to be disagreed with for us.
Ook wants to keep until we have an acceptable objective replacement
> 5796(RCDs)I personally disagree with, RCDs are useful and needed. It's only a crew nerf and a CE buff.
Ook doesn't seem to have any issues with RCDs in current state.  We can add this to the list of things to look at.
> 5851(Tilde fixes) No. No please. It's just going to be endless nya~ spam.
As a member of the catpeople coalition, NYAAAAAAAAAAAAAAA~~~~~~~~~~~~~~ uwu
> 5574(Viro updates) Generally good, but: *Self-respiration no longer causes farting. *Teratomas cannot spawn from a brain-dead person. *Wizarditus no longer gives real clothing.
Only one I do agree with afaik is the terratomas/braindead.... seems like a decent fix, I don't really care about wizarditus giving actual wizard clothing, the likelihood of it being relevant leads me to believe that it's more of a silly coincidence.  and farts are a meme who cares

**Pull request merged**